Home task:HTML 01 first part

Artem781 opened the pull request.

pluhin commented on anaron/Decision_1.2.html:
```diff
@@ -0,0 +1,66 @@
+    <p><em>Не забудьте поставить символ табуляции перед строкой «print cntr».</em> Это важно. В
+        Python для указания на блоки кода используются не круглые «(» и фигурные «{» скобки
+        как в других языках программирования, а отступы.</p>
+
```
Need to remove this extra lines

pluhin commented on anaron/Decision_1.3.html:
```diff
@@ -0,0 +1,72 @@
+    </p>
+    <p>Комментарии можно вставлять в любое место в строке, но помните, что Python
+        проигнорирует всё, что идёт после '#'.</p>
+
```
Empty lines  nee d remove

pluhin commented on anaron/Decision_1.3.html:
```diff
@@ -0,0 +1,72 @@
+        комментарии станут важными. Также при их помощи вы заставите Python игнорировать
+        какие-либо части кода. Чтобы закомментировать строку, поставьте перед ней знак '#'.
+        Например:</p>
+    #
```
\# - also code, need to shift into pre tag

pluhin commented on anaron/Decision_1.html:
```diff
@@ -0,0 +1,23 @@
+    <H1>Это заголовок</H1>
+    <P><DEL>Добрый день, <EM>Уважаемые слушатели</EM></DEL>! Сегодня мы с вами будем рассматривать структуру стандартного <STRONG>HTML</STRONG> файла.</P>
+    <P>Далее мы будем использовать произвольный текст для наполнения контентом. Не стоит вдумываться в его содержание.</P>
+    <P>«Си́мпсоны» (англ. The Simpsons) — самый длинный мультсериал в истории американского телевидения, длящийся 25 сезонов. Первая мини-серия «Good night» была показана в «Шоу Трейси Ульман» 19 апреля 1987 года.
```
This tag p wasn't closed

Artem781 commented:
decision_1-fixed- In lines №15 I closed tag P(</P>),
decision_1.2-fixed-In lines №54 I deleted empty lines,
decision_1.3-fixed-In lines №53 i put element # in tag <pre>,
I corrected task accoding to your comments. Please cheak it

Artem781 commented:
decision_1-fixed- In lines №15 I closed tag P(</P>),
decision_1.2-fixed-In lines №54 I deleted empty lines,
decision_1.3-fixed-In lines №53 i put element # in tag <pre>,
I corrected task accoding to your comments. Please cheak it

pluhin commented on anaron/02. HTML. Table/Decision_2_PytonPart3.html:
```diff
@@ -0,0 +1,94 @@
+<thml>
+    <head>
+    </head>
+    <meta charset="UTF-8">
```
this two tags should be inside tag head

Artem781 replied:
thank. I corrected this error.

pluhin commented on anaron/02. HTML. Table/site.by/about.html:
```diff
@@ -0,0 +1,35 @@
+    <H1>about</H1>
+    <h2>Меню с вложенным списком</h2>
+    <ul>
+        <li><a href="about.html">about</a></li>
```
this li tag should be contain whole sub list, not just tag a, look carefully our slides

pluhin commented on anaron/02. HTML. Table/site.by/about.html:
```diff
@@ -0,0 +1,35 @@
+            <li><a href="about/contacts.html">contacts</a></li>
+            <li><a href="about/scheme.html">scheme</a></li>
+        </ul>
+        <li><a href="catalog.html">catalog</a></li>
```
same as above

pluhin commented on anaron/02. HTML. Table/site.by/about.html:
```diff
@@ -0,0 +1,35 @@
+            <li><a href="catalog/a_w.html">a_w</a></li>
+            <li><a href="catalog/s_s.html">s_s</a></li>
+        </ul>
+        <li><a href="news/news.html">news</a></li>
```
same as above

pluhin commented on anaron/02. HTML. Table/site.by/about.html:
```diff
@@ -0,0 +1,35 @@
+        <ul>
+            <li><a href="news/news.html">news</a></li>
+        </ul>
+        <li><a href="promo.html">promo</a></li>
```
same as above

pluhin commented on anaron/02. HTML. Table/site.by/about/contacts.html:
```diff
@@ -0,0 +1,34 @@
+<BODY>
+    <H1>contacts</H1>
+    <ul>
+        <li><a href="../about.html">about</a></li>
```
wrong sublist construction
